### DISCUSSION TITLE
Isolating Users on a Single MinIO Server

### DISCUSSION THREAD

**Discussion by noobed-max** [minio/minio#20241] [Q&A]
new to this, I'm working on a project with MinIO and need to set up isolated environments for different user clients. The goal is to allow each user to create and manage their own buckets but also give them the ability to create and manage their own policies and groups while being isolated/hidden from other users and groups in the same server.

in summary:

- Allow this user to create and manage their own buckets which can be seen only by them

- Enable the user to create their own groups and policies

- Allow the user to create and manage their own sub-users

Is this possible? if not is there a way to implement this?

also if the approach i am taking is not good, can i know your POV

**Comment by mr4x2** [comment]
same question
